package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "etherspot",
-  "version": "1.35.2",
+  "version": "1.37.0",
```

Comment:
shouldn't it be 1.36 as the next version available?

Reply:
Okay got it so pls merge it after hop integration

Reply:
If Hop is merged first then (which is 1.36.0) then this should follow on from it